tests/unit/UsersControllerTest.php

```diff
@@ -1777,6 +1779,81 @@ public function testCreateSuccessfulWithValidEmailAdmin() {
 		$this->assertEquals(Http::STATUS_CREATED, $response->getStatus());
 	}
 
+	public function testCreateSuccessfulWithEmailAndUsername() {
+		$this->mailer->expects($this->once())
+			->method('validateMailAddress')
+			->willReturn(true);
+
+		$iUser = $this->createMock(IUser::class);
+
+		$this->userSession->expects($this->exactly(2))
+			->method('getUser')
+			->willReturn($iUser);
+
+		$this->groupManager
+			->expects($this->any())
+			->method('isAdmin')
+			->will($this->returnValue(true));
+
+		$this->userManager->expects($this->once())
+			->method('userExists')
+			->willReturn(false);
+
+		$this->eventDispatcher->expects($this->once())
+			->method('dispatch')
+			->with('OCP\User::createPassword', new GenericEvent());
+
+		$this->secureRandom->expects($this->exactly(2))
+			->method('generate')
+			->willReturn('AsDfGh12345');
+
+		$targetUser = $this->createMock(User::class);
+		$targetUser->expects($this->any())
```

Comment:
you can drop expects(any()) - this is the default

Reply:
Removed `expects($this->any())`. Thanks.